src/Widgets/LayoutWidget.vala

```diff
@@ -54,14 +40,14 @@ public class LayoutWidget : Gtk.Grid {
     }
 
     public override bool draw (Cairo.Context cr) {
-        var scale_factor = get_scale_factor ();
         gkbd_drawing.render (cr,
         Pango.cairo_create_layout (cr), 0, 0,
             get_allocated_width (),
             get_allocated_height (),
-            scale_factor,
-            scale_factor
+            50,
```

Comment:
Any explanation for this change?

Reply:
This is DPI, not scale factor and is hardcoded in the prebuilt dialog version of GTK's keyboard widget, so this just matches.